packages/utilities/src/test/reactTestRenderer.ts

```diff
@@ -0,0 +1,14 @@
+import { act, TestRendererOptions, ReactTestRenderer, create as defaultCreate } from 'react-test-renderer';
+import { ReactElement } from 'react';
+
+/**
+ * Wrapping `create` from `react-test-renderer' with `act`.
+ */
+export function create(nextElement: ReactElement, options?: TestRendererOptions): ReactTestRenderer {
```

Comment:
Nit: you can make this more "future-proof" by using the typings of `defaultCreate`

```ts
export function create(...args: Parameters<typeof defaultCreate>): ReturnType<typeof defaultCreate>{
  let component: ReturnType<typeof defaultCreate>;
  act(() => {
    component = defaultCreate(...args);
  });

  return component!;
}

Reply:
great suggestion, fixed!